proto/sentry_protos/snuba/v1/endpoint_delete_trace_items.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+// import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+// DeleteTraceItemsRequest initiates a delete of all matching items in a trace.
+// It should also remove downsampled copies.
```

Comment:
Can you specify we will either delete the `trace_ids` or the `filter` conditions, but not both at the same time?